pkgs/build-support/cc-wrapper/default.nix

```diff
@@ -115,7 +109,7 @@ stdenv.mkDerivation {
     # Binutils, and Apple's "cctools"; "bintools" as an attempt to find an
     # unused middle-ground name that evokes both.
     inherit bintools;
-    inherit libc nativeTools nativeLibc nativePrefix isGNU isClang default_cxx_stdlib_compile;
```

Comment:
This may be used outside by wrappers outside of nixpkgs. might be worth keeping, but just as meta-information.

Reply:
It might be, but I think that's bad :)

Reply:
There's no good way to do use libraries that want these c flags in without some meta-info like this. For instance bindgen in rust or anything that uses libclang. If we're going to be removing it, we should at least have a replacement.

Reply:
see https://github.com/NixOS/nixpkgs/pull/26474 for why this was exported

Reply:
Right I would rather have a more general way to get flags and just this one in particular. The default part (i.e. miy get overriden) is no good too.